drivers/windows/dir_access_windows.cpp

```diff
@@ -200,9 +200,9 @@ String DirAccessWindows::get_current_dir(bool p_include_drive) const {
 		return current_dir;
 	} else {
 		if (_get_root_string().is_empty()) {
-			int p = current_dir.find(":");
-			if (p != -1) {
```

Comment:
It's probably not very good design in the first place for us to have class members named `p` or `f` in `DirAccess` and `FileAccess`. Might warrant further refactoring (in another PR).